crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -32,6 +32,21 @@ def role_constraints
     end
   end
 
+  def shutdown_openstack_at_nodes
+    proposal = ProposalObject.find_proposal('crowbar', 'default')
+
+    NodeObject.all.each do |node|
+      if node.state == "crowbar_upgrade"
```

Comment:
Style/Next: Use `next` to skip iteration. (https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)